agent/stats/utils_test.go

```diff
@@ -38,6 +38,7 @@ func TestIsNetworkStatsError(t *testing.T) {
 func TestDockerStatsToContainerStatsCpuUsage(t *testing.T) {
 	// doing this with json makes me sad, but is the easiest way to deal with
 	// the inner structs
+	numCores = 4
```

Comment:
Can you add a note that this is changing a package-level variable?  It's hiding here like a locally-scoped variable and is non-obvious that it matters.